.github/workflows/ci.yaml

```diff
@@ -54,22 +58,23 @@ jobs:
     - uses: actions/checkout@v4
       with:
         path: QuantLib-Risks-Cpp
-    - name: Setup
-      run: |
-        sudo apt update
-        sudo apt install -y libboost-dev ccache ninja-build
     - name: ccache
       uses: hendrikmuhs/ccache-action@v1.2.12
       with:
         key: linux-${{ matrix.disable_aad }}
         max-size: 650M
+    - name: Setup
+      run: |
+        apt-get update \
+        && apt install -y libboost-dev ccache ninja-build
```

Comment:
Isn't boost already installed in the container?

Reply:
It is with the rolling container, I added it back by accident. Will fix.